### ISSUE TITLE
Laravel 9 Support?

### ISSUE THREAD

**Issue by seabasss** [glhd/laravel-addressing#37]
Is it possible to add Laravel 9 support? Unless this is because of something else?

```
 Problem 1
    - illuminate/validation[v7.0.0, ..., v7.28.4] require php ^7.2.5 -> your php version (8.1.2) does not satisfy that requirement.
    - illuminate/validation[v8.6.0, ..., v8.11.2] require php ^7.3 -> your php version (8.1.2) does not satisfy that requirement.
    - Root composer.json requires galahad/laravel-addressing ^3.0 -> satisfiable by galahad/laravel-addressing[3.0.0].
    - Conclusion: don't install laravel/framework v9.0.0-beta.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.3 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.4 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.5 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.1 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.1.0 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.1 (conflict analysis result)
    - galahad/laravel-addressing 3.0.0 requires illuminate/validation ^v7.0|^v8.0 -> satisfiable by illuminate/validation[v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev].
    - Only one of these can be installed: illuminate/validation[v7.0.0, ..., 7.x-dev, v8.6.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev], laravel/framework[v9.0.0-beta.1, ..., 9.x-dev]. laravel/framework replaces illuminate/validation and thus cannot coexist with it.
    - Root composer.json requires laravel/framework ^9.0 -> satisfiable by laravel/framework[v9.0.0-beta.1, ..., 9.x-dev].
 ```

Thanks!

**Comment by inxilpro** [comment]
We just released `3.1.0` with PHP 8 and Laravel 9 support!

**Comment by seabasss** [comment]
Awesome! Thanks
